packages/dropdown-react/src/Select.tsx

```diff
@@ -20,23 +27,53 @@ export function Select({
     inline = false,
     helpLabel,
     errorLabel,
+    variant,
+    placeholder,
+    value,
```

Comment:
I do not know if I like it, but you could set value to a default parameter:
```
value = placeholder || getSelectValuePairFrom(items[0]).value
```

And remove the `if(value)`-stuff.

Reply:
I discovered we also need a check for if there are any items, so it gets a bit complicated for inlining, imo.